System/Console/ANSI/Example.hs

```diff
@@ -264,4 +266,121 @@ titleExample = do
     pause
     -- Yup, I'm a new title! - ansi-terminal-ex - 83x70
     ---------------------------------------------------
-    -- Title Demo
\ No newline at end of file
+    -- Title Demo
+
+codesExample :: IO ()
+codesExample = do
```

Comment:
I agree that when all Windows terminals are assumed to be not ANSI-aware, this example is dull on both Windows and non-Windows systems. I was thinking ahead to when a Windows terminal might well be ANSI-aware and added this example to assist with my testing (to check that the `...Code` functions were indeed returning the ANSI codes on ANSI-aware Windows terminals).